### ISSUE TITLE
jump boost + shrinking incompatibility

### ISSUE THREAD

**Issue by xkforce** [CammiePone/Resize-Potion#9]
If the player has n levels of shrinking, applying less than n-1 levels of jump boost does not increase jump height.  If the player has jump boost at higher levels than that then the player's jump height actually goes down when jump boost is active.  eg.  shrinking 3 + jump boost 1 does not increase jump height; 2+ reduces jump height.  shrinking 4 + jump boost 2 does not increase jump height; 3+ reduces jump height etc.

**Comment by CammiePone** [comment]
That's probably due to how I had to increase jump height, and how jump boost is handled. Unlike literally every other potion, jump boost isn't determined by an entity attribute. It's hard coded, and it kinda sucks.

**Comment by xkforce** [comment]
Good old Mojang hard coding things to ruin our fun again.

This was something I ran into while testing different potion analogues of potion of the turtle master from 1.13 where you combine a high level good effect with a bad one and I wanted to see what happened if I combined one based on a high level jump boost potion with one based on a high level shrinking potion.  Good news is that the other one I tried with a high level speed potion worked wonderfully.
